OMCompiler/Compiler/Template/CodegenFMUCpp.tpl

```diff
@@ -794,6 +794,10 @@ case SIMCODE(modelInfo=MODELINFO(__), makefileParams=MAKEFILE_PARAMS(__), simula
   # link static libs to avoid dependencies; can't link all static under Linux
   else ifeq ($(findstring gcc,$(CC)),gcc)
     $(eval LIBS=$(LIBS) $(if $(findstring linux,$(PLATFORM)),-static-libstdc++ -static-libgcc,-static))
+    $(eval LIBS=$(LIBS) $(if $(findstring win,$(PLATFORM)),-static-libstdc++ -static-libgcc,-static))
```

Comment:
The second line with `$(findstring win` should not be needed as it was working with gcc before.

Reply:
Yeah, the second line actually added -static on Linux which broke some tests.